app/static/src/app/components/fit/FitApp.vue

```diff
@@ -9,12 +9,12 @@
           <code-tab></code-tab>
         </template>
         <template v-slot:Options>
-          <options-tab></options-tab>
+          <options-tab :fit-tab-is-open="fitTabIsOpen"></options-tab>
         </template>
       </wodin-tabs>
     </template>
     <template v-slot:right>
-      <wodin-tabs id="right-tabs" :tabNames="['Run', 'Fit', 'Sensitivity']">
+      <wodin-tabs id="right-tabs" :tabNames="['Run', 'Fit', 'Sensitivity']" @tabSelected="rightTabSelected">
```

Comment:
I think selectedTabs should be added to the state, so as to avoid props.

Reply:
I think it's ok in general for components to know their own state via props like this to avoid cluttering up the store with presentation-specific state. But on the other hand, it would simplify things a little here to keep selected tab in the store - and we're also going to need it for the Sensitivity-related stuff. So, yes, I'll add something like selectedVisuationTab to AppState.